fdbserver/worker.actor.cpp

```diff
@@ -1739,6 +1744,55 @@ ACTOR Future<Void> updateClusterId(UID ccClusterId, Reference<AsyncVar<Optional<
 	return Void();
 }
 
+ACTOR Future<Void> cleanupStaleDiskStore(Reference<AsyncVar<ServerDBInfo>> dbInfo,
+                                         Reference<CommitProxyInfo> commitProxies,
```

Comment:
why not using the commit proxies in `dbInfo`? If there is a recovery, commit proxies can change.

Reply:
It seems `DiskStore` is not cleaned in this function.